api/apps/geoprocessing/src/marxan-sandboxed-runner/adapters-blm/marxan-sandbox-blm-runner.service.ts

```diff
@@ -16,7 +16,7 @@ export class MarxanSandboxBlmRunnerService
   readonly #controllers: Record<string, AbortController> = {};
 
   constructor(
-    private readonly inputFilesHandler: BlmInputFiles,
+    private readonly inputFilesHandlerFactory: BlmInputFilesFactory,
```

Comment:
That's what we had before, we even changed de scope so it wouldn't be a Singleton but didn't work

Reply:
@AaronPerezPerez the service using request-scoped injectable would also need to be request-scoped - otherwise, BlmRunnerService requests it.... once. Thats why there were `moduleRef.create(...)`